### PR TITLE
Add edid override for AYANEO GEEK.

### PR DESCRIPTION
**Pull request by oliverchang**
This fixes an edid parse failure by patching out a bad section in the edid, and seems to fix
https://github.com/ValveSoftware/gamescope/issues/864.

### CODE REVIEW

**Review by Samsagax — CHANGES_REQUESTED** [review]

**Comment by oliverchang** [comment]
Note there are AYANEO GEEK devices out there with a 1200p panel (I believe the same as the AYANEO 2) instead of the default 800p panel which I have. I'm not sure how to distinguish those at this point though.

Perhaps we could check the original edid shasum and only patch in that case? Is that available in the context where this script runs?

File: usr/share/device-quirks/scripts/ayaneo/ayaneo_table.sh
Comment (oliverchang) [reply usr/share/device-quirks/scripts/ayaneo/ayaneo_table.sh:18]: Hmm, I'm doing a direct match here using ==. Is that still necessary?

**Comment by oliverchang** [comment]
CC @ruineka too :)

File: usr/share/device-quirks/scripts/ayaneo/ayaneo_table.sh
Comment (pastaq) [reply usr/share/device-quirks/scripts/ayaneo/ayaneo_table.sh:18]: If we're going to do this then I'd prefer to have a PRODUCT_LIST_GEEK and remove geek from the 2 list. Add the audio quirk to the geek.sh the 2.sh has.

**Comment by ruineka** [comment]
We might be having a proper fix underway soon. Don't merge this unless we have too.

**Comment by oliverchang** [comment]
https://gitlab.freedesktop.org/emersion/libdisplay-info/-/merge_requests/164 is the code workaround that will fix this particular instance.

**Comment by ruineka** [comment]
> https://gitlab.freedesktop.org/emersion/libdisplay-info/-/merge_requests/164 is the code workaround that will fix this particular instance.

Indeed it is, any chance you can comment under there where it's asking about permission to list the EDID under MIT? I added my EDID as well but they used yours as the sample to be used.

**Comment by pastaq** [comment]
has there been any movement on the patch for this, or a way to ID the 800P display units only?

**Pull request closed without merging**
